client/my-sites/email/gsuite-purchase-cta/style.scss

```diff
@@ -36,7 +36,7 @@
 
 	@include breakpoint( '>1040px' ) {
 		flex-basis: 100%;
-		max-width: 400px;
+		max-width: 410px;
```

Comment:
This minor change is just to prevent the main heading from wrapping, at least in English (tested on Chrome on Windows):

Before | After
------ | -----
![04](https://user-images.githubusercontent.com/594356/69557092-4c75d580-0fa6-11ea-80f2-eb7efc8fb5d6.png) | ![01](https://user-images.githubusercontent.com/594356/69556382-e5a3ec80-0fa4-11ea-8f89-9812d282b6d9.png)